packer/ansible/roles/isc-dhcp-server/files/systemd-isc-dhcp-server

```diff
@@ -18,4 +18,5 @@
 
 # On what interfaces should the DHCP server (dhcpd) serve DHCP requests?
 #	Separate multiple interfaces with spaces, e.g. "eth0 eth1".
-INTERFACES="enp0s8"
+#   for 16.04, either ens33(e1000) or ens192(vmxnet3)
+INTERFACES="ens33 ens192"
```

Comment:
I am wondering if there are better ways to get the interface name than use the hard-cod-ed name. Any possibility that we inspect the syslog/dmesg and get the name?

```
onrack@~/github/on-static> dmesg | grep e1000
[    1.019679] e1000: Intel(R) PRO/1000 Network Driver - version 7.3.21-k8-NAPI
[    1.019680] e1000: Copyright (c) 1999-2006 Intel Corporation.
[    1.367837] e1000 0000:02:00.0 eth0: (PCI:66MHz:32-bit) 00:50:56:aa:bc:ab
[    1.367843] e1000 0000:02:00.0 eth0: Intel(R) PRO/1000 Network Connection
[    1.711464] e1000 0000:02:01.0 eth1: (PCI:66MHz:32-bit) 00:50:56:aa:66:16
[    1.711468] e1000 0000:02:01.0 eth1: Intel(R) PRO/1000 Network Connection
[    2.054803] e1000 0000:02:02.0 eth2: (PCI:66MHz:32-bit) 00:50:56:aa:11:76
[    2.054807] e1000 0000:02:02.0 eth2: Intel(R) PRO/1000 Network Connection
[    2.398718] e1000 0000:02:03.0 eth3: (PCI:66MHz:32-bit) 00:50:56:aa:8b:33
[    2.398723] e1000 0000:02:03.0 eth3: Intel(R) PRO/1000 Network Connection
[    5.054553] e1000: eth2 NIC Link is Up 1000 Mbps Full Duplex, Flow Control: None
[    5.142130] e1000: eth3 NIC Link is Up 1000 Mbps Full Duplex, Flow Control: None
[    5.163323] e1000: eth0 NIC Link is Up 1000 Mbps Full Duplex, Flow Control: None
[    5.223507] e1000: eth1 NIC Link is Up 1000 Mbps Full Duplex, Flow Control: None
```